tests/robot-tests/tests/admin_and_public/bau/publication_owner.robot

```diff
@@ -0,0 +1,340 @@
+    user clicks radio   Approved for publication
+    user enters text into element  id:releaseStatusForm-internalReleaseNote  Approved by publication owner
+    user clicks radio   As soon as possible
+    user clicks button   Update status
+
+Wait for release process status to be Complete
+    [Tags]  HappyPath
+    user waits for release process status to be  Complete    ${release_complete_wait}
+    user reloads page  # EES-1448
+    user checks page does not contain button  Edit release status
+
+Navigate to administration as bau1 to remove approver access
+    [Tags]  HappyPath
+    user signs in as bau1
+    user goes to url  %{ADMIN_URL}/administration/users
+
+Navigate to manage users
+    [Tags]  HappyPath
+    user waits until element is enabled  //*[tbody]//tr[1]//td[3]//a
+    user clicks element  //*[tbody]//tr[1]//td[3]//a
+    user waits until page does not contain loading spinner
+    
+Give release contributor access to Analyst1
+    [Tags]  HappyPath
+    Sleep  0.5
+    user waits until page does not contain loading spinner
+    user scrolls to element  //*[text()="Release access"]
+    user waits until element is enabled  css:[name="selectedReleaseId"]
+    user selects from list by label  css:[name="selectedReleaseId"]  ${RELEASE_NAME}
+    user waits until element is enabled  css:[name="selectedReleaseRole"]
+    user selects from list by label  css:[name="selectedReleaseRole"]  Contributor
+    user clicks button  Add release access 
+    
+Remove release approver access from Analyst1
+    [Tags]  HappyPath
+    Sleep  0.5
+    user scrolls to element  //*[@data-testid="releaseAccessApprover"]    
+    user clicks testid element  releaseAccessApprover
+
+Check release contributor can make an amendment
+    [Tags]  HappyPath
+    user clicks link  Home
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}
+    user opens accordion section  ${PUBLICATION_NAME}
+    ${accordion}=  user gets accordion section content element  ${PUBLICATION_NAME}
+    ${details}=  user gets details content element  ${RELEASE_TYPE} (Live - Latest release)  ${accordion}
+    user waits until parent contains element   ${details}   xpath:.//a[text()="View this release"]
+    user clicks button  Amend this release  ${details}
+    user clicks button  Confirm
+
+Navigate to 'Footnotes' section
+    [Tags]  HappyPath
+    user clicks link  Footnotes
+    user waits until h2 is visible  Footnotes
+
+Add a Footnote as a release contributor
+    [Tags]  HappyPath
+    user waits until page contains link   Create footnote
+    user clicks link  Create footnote
+    user waits until h2 is visible  Create footnote
+    user clicks footnote radio   ${SUBJECT_NAME}   Applies to all data
+    user clicks element  id:footnoteForm-content
+    user enters text into element  id:footnoteForm-content  Footnote 3 ${FOOTNOTE_TEXT_3}
```

Comment:
I wouldn't have the footnote text be half hard-coded and half a variable.

My preference would be to hardcode the footnote text here and delete the variable, since we're not checking elsewhere that the footnote has been successfully created